shared/src/main/scala/Context.scala

```diff
@@ -27,8 +29,8 @@ import scala.io.StdIn
   */
 class Context private (
     private var stack: mut.ArrayBuffer[VAny],
-    private var _contextVarN: Option[VAny] = None,
-    private var _contextVarM: Option[VAny] = None,
+    private var _contextVarPrimary: Option[VAny] = None,
```

Comment:
Great idea making them more visually distinctive. Do we want to call them `ctxVarPrimary/Secondary` so it's a little shorter?

Reply:
I think that would be fine